apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -2,39 +2,57 @@
 // See LICENSE in the project root for license information.
 
 import * as child_process from 'child_process';
+import * as colors from 'colors';
+import { Executable } from '@microsoft/node-core-library';
+
+const DEFAULT_BRANCH: string = 'master';
+const DEFAULT_REMOTE: string = 'origin';
 
 export class VersionControl {
-  public static getChangedFolders(targetBranch?: string): Array<string | undefined> | undefined {
-    const branchName: string = targetBranch ? targetBranch : 'origin/master';
-    const output: string | undefined = child_process.execSync(`git diff ${branchName}... --dirstat=files,0`)
-      .toString();
-    return output.split('\n').map(s => {
-      if (s) {
-        const delimiterIndex: number = s.indexOf('%');
-        if (delimiterIndex > 0 && delimiterIndex + 1 < s.length) {
-          return s.substring(delimiterIndex + 1).trim();
+  public static getChangedFolders(targetBranch: string): Array<string | undefined> | undefined {
+    const output: string = child_process.execSync(`git diff ${targetBranch}... --dirstat=files,0`).toString();
+    const fetchResult: boolean = VersionControl._tryFetchRemoteBranch(targetBranch);
+    if (!fetchResult) {
```

Comment:
Seems like `_tryFetchRemoteBranch()` can be skipped if we're comparing against **origin/master** (i.e. the PR is not coming from a forked GitHub repo).  Suppose the person merged/cherry-picked changes from **origin/master** into their PR branch. 
- **If those integrations were made locally**, they must have been pulled down (because we assume people don't commit directly to master), so **origin/master** must be recent enough to have those changes;  OR
- **If those integrations were made on the remote server**, in order for them to be in our PR branch, we must have fetched from the origin.  And fetching would have brought along the status of **origin/master** along with our branch.

Reply:
That's probably a reasonable assumption.